lib/sum_of_multiples.rb

```diff
@@ -0,0 +1,30 @@
+class SumOfMultiples
+
+  attr_reader :numbers
+
+  def initialize(*numbers)
+    @numbers = numbers
+  end
+
+  def sum_of_multiples(range)
+    if @numbers.length == 0 || @numbers == [0]
```

Comment:
Avoid if-else. It reduces readability from the code. Follow this
https://williamdurand.fr/2013/06/03/object-calisthenics/